src/pages/home.js

```diff
@@ -36,27 +36,28 @@ export const Home = ({
   } else {
     recentPostMarkup = <p>Loading...</p>;
   }
-  let createPostMarkup;
-  if (!isAuthenticated) {
-    createPostMarkup = (
-      <Link to='/login'>
-        <Button
-          variant='contained'
-          className={classes.createButton}
-          color='primary'
-        >
-          <AddIcon className={classes.extendedIcon} />
-          Create Post
-        </Button>
-      </Link>
-    );
-  } else {
-    createPostMarkup = <NewPost />;
-  }
+  const makeCreatePostEditor = () => {
+    if (isAuthenticated) {
+      return <NewPost />;
+    } else {
```

Comment:
You don't actually need the else{}, you can just return <Link ... after this if statement closes.

Reply:
Fixed. Will probably need to go back and refactor some of these pages at some point.